src/Types.ts

```diff
@@ -96,6 +96,11 @@ export interface IUnspentTxOutput
      * The bytes of lock script in transaction output
      */
     lock_bytes: string;
+
+    /**
+     * Does the value exist in the UTXO table?
+     */
+    exist?: boolean
```

Comment:
The question is strange.
`Whether to include UTXO table`

Reply:
It was removed and treated as returning an array of utxo.